bibtex-actions.el

```diff
@@ -136,15 +133,22 @@ key associated with each one."
    (let* ((pdf (if (assoc "=has-pdf=" (cdr candidate)) " has:pdf"))
           (note (if (assoc "=has-note=" (cdr candidate)) "has:note"))
           (link (if (assoc "doi" (cdr candidate)) "has:link"))
-          (add (s-trim-right (s-join " " (list pdf note link)))))
+          (citekey (bibtex-completion-get-value "=key=" candidate))
+          (add (s-trim-right (s-join " " (list pdf note link))))
+          ; TODO: add separation between target and suffix
+          (suffix (bibtex-completion-format-entry candidate (1- (frame-width))
```

Comment:
This, plus the defvar, are the only pieces of code dependent on the bibtex-completion PR. So could comment out, and add a fixed placeholder for now.